src/modules/Builder/features/ActivityItems/ItemConfiguration/Alerts/Alert/Alert.tsx

```diff
@@ -0,0 +1,53 @@
+import { useTranslation, Trans } from 'react-i18next';
+import { useFormContext } from 'react-hook-form';
+
+import { Svg } from 'shared/components';
+import { InputController, SelectController } from 'shared/components/FormComponents';
+import { StyledTitleBoldSmall, variables } from 'shared/styles';
+
+import { items, options } from './Alert.const';
+import { StyledAlert, StyledRow, StyledRemoveBtn, StyledDescription } from './Alert.styles';
+import { ItemConfigurationForm } from '../../ItemConfiguration.types';
+import { AlertProps } from './Alert.types';
+
+export const Alert = ({ index, removeAlert }: AlertProps) => {
+  const { t } = useTranslation('app');
+  const { control } = useFormContext<ItemConfigurationForm>();
+
+  // TODO: change fields depends on item type
```

Comment:
Pls implement that now, because we have all designs and all variations of alerts
And no need to create separate task in future and it could take more time

Reply:
do not have all specs yet